datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1685,4 +1632,234 @@ ZkAdapter getZkAdapter() {
   CoordinatorConfig getConfig() {
     return _config;
   }
+
+  /**
+   * Encapsulates metric registration and update for the {@link Coordinator}
+   */
+  private static class CoordinatorMetrics {
+    private static final String MODULE = Coordinator.class.getSimpleName();
+
+    private static final String NUM_RETRIES = "numRetries";
+    private static final String NUM_ERRORS = "numErrors";
+    private static final String HANDLE_EVENT_PREFIX = "handleEvent";
```

Comment:
I did that on purpose; the metric was broken (not being emitted since it wasn't returned from `getMetricInfos()`) and a dash isn't common in metric names. Please, let me know if you still think I should fix it. cc: @somandal

Reply:
You can ignore it if it is not emitted. My only concern was regression, which is not the case.